colcon_notification/event_handler/desktop_notification/__init__.py

```diff
@@ -33,7 +33,6 @@ def __init__(self):  # noqa: D107
         super().__init__()
         satisfies_version(
             EventHandlerExtensionPoint.EXTENSION_POINT_VERSION, '^1.0')
-        self._start_time = time.time()
```

Comment:
is that change on purpose? Or should this also be changed to `monotonic()`?

Reply:
I didn't see where this field was used anywhere in this class and there are no subclasses of it either. Hence I removed it.